nvchecker_source/alpm.py

```diff
@@ -0,0 +1,27 @@
+# MIT licensed
+# Copyright (c) 2020 DDoSolitary <DDoSolitary@gmail.com>, et al.
+
+from nvchecker.api import GetVersionError
+from pyalpm import Handle
+
+async def get_version(name, conf, **kwargs):
+  pkgname = conf.get('alpm', name)
+  dbpath = conf.get('dbpath', '/var/lib/pacman')
+  repo = conf.get('repo')
+  strip_release = conf.get('strip_release', False)
+  provided = conf.get('provided')
+  handle = Handle('/', dbpath)
+  db = handle.register_syncdb(repo, 0)
```

Comment:
Please cache the `db` or similar using the `cache` argument passed in.

Reply:
Done.

Reply:
Should I cache the result of `get_pkg`? It seems that it loops through the package cache to find the specified name. I'm not sure if it should be considered an expensive operation.

Reply:
Done.